docs/src/main/asciidoc/_elytron/components/Filesystem_Security_Realm.adoc

```diff
@@ -21,9 +23,11 @@ Filesystem security realm can be added with WildFly management CLI or with Elytr
 The following command creates realm with name `fsRealm` that resides in directory `fs-realm-users` inside the base configuration directory defined by `jboss.server.config.dir` property. The levels number is 2 and encoded value is true.
 The charset is UTF-8 and the string encoding is BASE64.
 
+The credential-store being referenced is called `mycredstore` and the alias of the `secret-key` in that credential-store is `key`. The secret key will be used to encrypt and decrypt the filesystem realm
```

Comment:
`credential-store` instead of credential-store (2 places on this line)

Should add a period at the end of realm.

Reply:
Changed